### ISSUE TITLE
The "--prefix" / "--suffix" option does not exist.

### ISSUE THREAD

**Issue by semspanhaak** [diego-rlima/artisan-make-file-location#5]
Hi there @diego-rlima ,

I've been looking for some time for a sollution like you got here, seems amazing! I am working with a custom module loader so your sollution is the bomb. I've tried installing it in my new 'boilerplate' project, please see: https://github.com/semspanhaak/module-loader-template, but without any succes. 

I keep getting the following errors, while I did follow your listed steps at https://github.com/diego-rlima/artisan-make-file-location#installation
<img width="820" alt="image" src="https://user-images.githubusercontent.com/11527464/90777396-a6b88280-e2fb-11ea-9354-308d7677b66d.png">

Do you know what is happening or is wrong?

Laravel Framework 7.25.0
PHP 7.4.9 (cli) (built: Aug  7 2020 19:23:06) ( NTS )
Zend Engine v3.4.0, Copyright (c) Zend Technologies
    with Xdebug v2.9.6, Copyright (c) 2002-2020, by Derick Rethans
    with Zend OPcache v7.4.9, Copyright (c), by Zend Technologies

**Comment by diego-rlima** [comment]
Hi @semspanhaak ,

I'm glad you liked the project.

I noticed that that the original commands were not being replaced by the one in the package. So I [released a fix](https://github.com/diego-rlima/artisan-make-file-location/releases/tag/1.5.1) for that.

Now it seems to work with Laravel 7.25.0 and PHP 7.4.3

**Note**
The package has not been updated for over 2 years, so other bugs may occur.
Any commands added to Laravel after version 5.6 are not supported by this package.

I didn't need to use it anymore, so I ended up leaving it out of date.
I would still like to update it, but I have no prediction of when this will happen. :(
